scripts/SREP/get-pull-secret-email/script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
The existing osdctl subcommand https://github.com/openshift/osdctl/blob/master/cmd/cluster/validatepullsecret.go#L56 will check the ocm registry credential as well. It seems we cannot do the same in managed scripts?

Reply:
For the OCM part we still need to do that in `osdctl`.